lib/ruby-push-notifications/version.rb

```diff
@@ -1,3 +1,3 @@
 module RubyPushNotifications
-  VERSION = "0.2.2"
+  VERSION = "0.3.0"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.